pom.xml

```diff
@@ -33,35 +33,240 @@
     <version>1.0.0-SNAPSHOT</version>
     <packaging>pom</packaging>
 
-    <modules>
-        <!-- Make sure to list support-githooks first then gitsetup since gitsetup has a dependency on it -->
-        <module>support-githooks</module>
-        <module>gitsetup</module>
-        <module>support-checkstyle</module>
-        <module>coverage-updater</module>
-        <module>support-findbugs</module>
-        <module>support-owasp</module>
-        <module>support-pmd</module>
-        <module>support-karaf</module>
-        <module>version-validation-plugin</module>
-    </modules>
-
     <properties>
-        <maven.assembly.plugin.version>2.2.2</maven.assembly.plugin.version>
+        <abdera.osgi.version>1.1.3_2</abdera.osgi.version>
```

Comment:
❓I like the alphabetical order, but I'm wondering if that's enough. Should we consider keeping the build/maven, documentation and test dependency versions separate?

Reply:
We can. I would assume most people will use the find in page/search functionality when making edits, but I could see your point about groups of dependencies being edited together. I can give it an attempt.